config/mini_games/speedrun.lua

```diff
@@ -0,0 +1,115 @@
+            'effectivity-module-2','effectivity-module'
+        },
+        items = {
+            'power-armor-mk2','automation-science-pack','logistic-science-pack','chemical-science-pack','military-science-pack',
+            'utility-science-pack','processing-unit','electronic-circuit','iron-plate','iron-ore',
+            'copper-cable','copper-plate','copper-ore','advanced-circuit','plastic-bar',
+            'coal','electric-engine-unit','engine-unit','steel-plate','iron-gear-wheel',
+            'pipe','low-density-structure','speed-module-2','speed-module','effectivity-module-2',
+            'effectivity-module','transport-belt','inserter','sulfur','piercing-rounds-magazine',
+            'firearm-magazine','grenade','stone-wall','stone-brick','stone',
+            'flying-robot-frame','battery'
+        },
+        entities = {
+            'steam-engine', 'lab', 'assembling-machine-1', 'assembling-machine-2', 'oil-refinery'
+        }
+    },
+    { --- Standard
+        goal = 'Launch one rocket from a silo, does not require a satellite.',
+        name = 'Standard',
+        rockets = 1,
+        research = {
+            'rocket-silo','concrete','advanced-material-processing','steel-processing','logistic-science-pack',
+            'automation-2','electronics','automation','speed-module-3','speed-module-2',
+            'speed-module','modules','advanced-electronics','plastics','oil-processing',
+            'fluid-handling','engine','advanced-electronics-2','chemical-science-pack','sulfur-processing',
+            'production-science-pack','productivity-module','advanced-material-processing-2','railway','logistics-2',
+            'logistics','productivity-module-3','productivity-module-2','rocket-fuel','flammables',
+            'advanced-oil-processing','rocket-control-unit','utility-science-pack','robotics','electric-engine',
+            'lubricant','battery','low-density-structure'
+        },
+        items = {
+            'rocket-silo','rocket-control-unit','low-density-structure','rocket-fuel','automation-science-pack',
+            'logistic-science-pack','chemical-science-pack','production-science-pack','utility-science-pack','steel-plate',
+            'iron-plate','iron-ore','processing-unit','electronic-circuit','copper-cable',
+            'copper-plate','copper-ore','advanced-circuit','plastic-bar','coal',
+            'electric-engine-unit','engine-unit','iron-gear-wheel','pipe','concrete',
+            'stone-brick','stone','speed-module','solid-fuel','transport-belt',
+            'inserter','sulfur','rail','iron-stick','electric-furnace',
+            'productivity-module','flying-robot-frame','battery'
+        },
+        entities = {
+            'steam-engine', 'lab', 'assembling-machine-1', 'assembling-machine-2', 'oil-refinery', 'rocket-silo'
+        }
+    },
+    { --- Marathon
+        goal = 'Launch ten rockets from a silo, does not require satellites.',
```

Comment:
No this is correct, lauching a rocket to complete the speedrun does not require you to launch a satellite. Although, it would be interesting to make Marathon require it so that its more than what Sandard requires.

Reply:
Satellites have been added to marathon see below.